src/main.jsx

```diff
@@ -31,10 +32,21 @@ const router = createBrowserRouter([
   },
 ]);
 
+const initLanguage = (lang) => {
+  if (lang === 'no') {
```

Comment:
Should probably use 'nb' instead as 'no' referst to norwegian as a whole. When there is the distiction between nynorsk and bokmål, I think we should use 'nn' and 'nb' respectively